[2025-04-04T18:57:15.275+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-04T18:57:15.315+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: google_maps_reviews_pipeline.Load_reviews1 manual__2025-04-04T18:56:57.108267+00:00 [queued]>
[2025-04-04T18:57:15.339+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: google_maps_reviews_pipeline.Load_reviews1 manual__2025-04-04T18:56:57.108267+00:00 [queued]>
[2025-04-04T18:57:15.358+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-04T18:57:15.393+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): Load_reviews1> on 2025-04-04 18:56:57.108267+00:00
[2025-04-04T18:57:15.409+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=336) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-04T18:57:15.414+0000] {standard_task_runner.py:72} INFO - Started process 351 to run task
[2025-04-04T18:57:15.419+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'google_maps_reviews_pipeline', 'Load_reviews1', 'manual__2025-04-04T18:56:57.108267+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/***dag.py', '--cfg-path', '/tmp/tmpkcmehekz']
[2025-04-04T18:57:15.421+0000] {standard_task_runner.py:105} INFO - Job 31: Subtask Load_reviews1
[2025-04-04T18:57:15.505+0000] {task_command.py:467} INFO - Running <TaskInstance: google_maps_reviews_pipeline.Load_reviews1 manual__2025-04-04T18:56:57.108267+00:00 [running]> on host c3eecb2684dd
[2025-04-04T18:57:15.689+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='google_maps_reviews_pipeline' AIRFLOW_CTX_TASK_ID='Load_reviews1' AIRFLOW_CTX_EXECUTION_DATE='2025-04-04T18:56:57.108267+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-04T18:56:57.108267+00:00'
[2025-04-04T18:57:15.692+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-04-04T18:57:15.693+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-04-04T18:57:15.694+0000] {logging_mixin.py:190} INFO - Current task name:Load_reviews1 state:running start_date:2025-04-04 18:57:15.318377+00:00
[2025-04-04T18:57:15.695+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews_pipeline and current dag run status:running
[2025-04-04T18:57:15.696+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-04T18:57:15.732+0000] {logging_mixin.py:190} INFO - Current directory: /opt/***/dags
[2025-04-04T18:57:15.734+0000] {logging_mixin.py:190} INFO - Data path: /opt/***/dags/input/reviews.csv
[2025-04-04T18:57:15.771+0000] {base.py:84} INFO - Retrieving connection 'postgres_defaut'
[2025-04-04T18:57:15.828+0000] {logging_mixin.py:190} INFO - Data stored in PostgreSQL.
[2025-04-04T18:57:15.830+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-04T18:57:15.850+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-04T18:57:15.851+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=google_maps_reviews_pipeline, task_id=Load_reviews1, run_id=manual__2025-04-04T18:56:57.108267+00:00, execution_date=20250404T185657, start_date=20250404T185715, end_date=20250404T185715
[2025-04-04T18:57:15.893+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-04-04T18:57:15.894+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-04-04T18:57:15.896+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews_pipeline queued_at:2025-04-04 18:56:57.160682+00:00
[2025-04-04T18:57:15.896+0000] {logging_mixin.py:190} INFO - Task hostname:c3eecb2684dd operator:PythonOperator
[2025-04-04T18:57:15.914+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-04T18:57:15.973+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-04T18:57:15.977+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
